test/scraped/response/decorator/clean_urls_test.rb

```diff
@@ -143,4 +143,18 @@ def link(selector)
   it 'ignores invalid mailto links' do
     page.broken_mailto_link.must_equal 'mailto:notanemail'
   end
+
+  describe 'AbsoluteUrls' do
+    it 'returns CleanUrls' do
+      check_warning = lambda do |msg|
+        msg.must_equal '`Scraped::Response::Decorator::AbsoluteUrls` has been deprecated. ' \
+          'Use `Scraped::Response::Decorator::CleanUrls` instead.'
+        nil
+      end
+
+      Scraped::Response::Decorator.stub(:warn, check_warning) do
+        Scraped::Response::Decorator::AbsoluteUrls.must_equal Scraped::Response::Decorator::CleanUrls
+      end
```

Comment:
I fear that this test is a little too far removed from the expected usage pattern to be sure that it's really testing the right thing.

I think it would be a lot simpler and clearer to show that declaring a class in the normal manner but using the old decorator, still works, but issues a warning.

That's a little bit tricky because it happens at the point of declaring the class, but something like this seems plausible:

```ruby
  describe 'AbsoluteUrls' do
    it 'gives a deprecation warning' do
      _out, err = capture_io do
        klass = Class.new(Scraped::HTML) do
          decorator Scraped::Response::Decorator::AbsoluteUrls

          field :relative_image do
            noko.at_css('.relative-image/@src').text
          end
        end
        klass.new(response: response).relative_image.must_equal 'http://example.com/person-123.jpg'
      end

      err.must_include 'has been deprecated'
    end
  end
```